packages/typeorm/README.md

```diff
@@ -0,0 +1,92 @@
+<p align="center">
+   <br/>
+   <a href="https://next-auth.js.org" target="_blank"><img width="150px" src="https://next-auth.js.org/img/logo/logo-sm.png" /></a>
+   <h3 align="center">NextAuth.js - <b>TypeORM Adapter</b></h3>
+   <p align="center">Authentication for Next.js</p>
+   <p align="center">
+   Open Source. Full Stack. Own Your Data.
+   </p>
+   <p align="center" style="align: center;">
+      <img src="https://github.com/nextauthjs/adapters/workflows/Build%20Test/badge.svg" alt="Build Test" />
+      <img src="https://github.com/nextauthjs/adapters/workflows/Integration%20Test/badge.svg" alt="Integration Test" />
+      <img src="https://img.shields.io/bundlephobia/minzip/@next-auth/typeorm-adapter" alt="Bundle Size"/>
+      <img src="https://img.shields.io/npm/dm/@next-auth/typeorm-adapter" alt="Downloads" />
+      <img src="https://img.shields.io/github/stars/nextauthjs/adapters" alt="Github Stars" />
+      <img src="https://img.shields.io/github/v/release/nextauthjs/adapters?include_prereleases" alt="Github Release" />
+   </p>
+</p>
+
+## Overview
+
+This is the TypeORM Adapter for [`next-auth`](https://next-auth.js.org). This package can only be used in conjunction with the primary `next-auth` package. It is not a standalone package.
+
+## Getting Started
+
+1. Install `next-auth` and `@next-auth/typeorm-adapter`
+```js
+npm install --save-prod next-auth @next-auth/typeorm-adapter
```

Comment:
Didn't actually know about the `--save-prod` flag. Looks like it is because it is the default behavior https://docs.npmjs.com/cli/v6/commands/npm-install and I have never had to use it. Could we remove it?

Reply:
Yeah its the same as `-s`, I just usually like to be explicit about these types of things in readmes. But seeing as its the default behavior anyway yeah lets just remove it.